src/data/codemodConfigSchema.ts

```diff
@@ -1,27 +1,59 @@
 import * as S from '@effect/schema/Schema';
 
+export const argumentsSchema = S.array(
+	S.union(
+		S.struct({
+			name: S.string,
+			kind: S.literal('string'),
+			default: S.union(S.string, S.undefined),
+		}),
+		S.struct({
+			name: S.string,
+			kind: S.literal('number'),
+			default: S.union(S.number, S.undefined),
+		}),
+		S.struct({
+			name: S.string,
+			kind: S.literal('boolean'),
+			default: S.union(S.boolean, S.undefined),
+		}),
+		// S.struct({
```

Comment:
should we keep it?

Reply:
i will uncomment it in  one of next pr when i add support of dropdowns